Exiled.API/Extensions/RagdollExtensions.cs

```diff
@@ -0,0 +1,51 @@
+// -----------------------------------------------------------------------
+// <copyright file="RagdollExtensions.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Extensions
+{
+    using System.Linq;
+
+    using Exiled.API.Features;
+
+    /// <summary>
+    /// A set of extensions for <see cref="Ragdoll"/>.
+    /// </summary>
+    public static class RagdollExtensions
+    {
+        /// <summary>
+        /// Finds and returns the <see cref="Room"/> the ragdoll is located in.
+        /// </summary>
+        /// <param name="ragdoll">The <see cref="Ragdoll"/> to check the room of.</param>
+        /// <returns>The <see cref="Room"/> the ragdoll is located in.</returns>
+        public static Room GetRoom(this Ragdoll ragdoll) =>
+            Map.FindParentRoom(ragdoll.gameObject);
+
+        /// <summary>
+        /// Returns the <see cref="RoleType"/> of the ragdoll.
+        /// </summary>
+        /// <param name="ragdoll">The <see cref="Ragdoll"/> to check the role of.</param>
+        /// <returns>The <see cref="RoleType"/> of the ragdoll.</returns>
+        public static RoleType Role(this Ragdoll ragdoll) =>
```

Comment:
follow only one convention, add the `Get` prefix to all methods that don't have one

Reply:
Gotcha